crates/core/src/client/constants.rs

```diff
@@ -118,3 +119,8 @@ lazy_static! {
     pub static ref COUNTER_CALL_TESTNET2: Call =
         StarknetCall { to: *COUNTER_ADDRESS_TESTNET2, selector: *INC_SELECTOR, calldata: vec![] }.into();
 }
+
+// This module contains constants to be used for deployment of Kakarot System
+lazy_static! {
+    pub static ref DEPLOY_FEE: FieldElement = FieldElement::from(100000_u64);
```

Comment:
why the deploy fee this amount?

Reply:
Just a random number we are using for now! We would have to estimate in future and then update it.

Reply:
but here it's a constant right? should we have an issue + a todo to track the fact that we'll make it dynamic?
it's being used in many places

Reply:
Oh, so the thing is that DEPLOY_FEE is storage variable on Kakarot main, see [here](https://github.com/kkrt-labs/kakarot/blob/9773e4d10a3c3a32fb8aa3cfbf6fdbff35d6985e/src/kakarot/library.cairo#L54).

So, we would need an off-chain service, updating this variable on periodic basis. The reason why it exists on the contract level is so that someone cannot drain an EOA by putting any random `DEPLOY_FEE`.

Reply:
Also, this constant is just in testing, and has no use in the working RPC binary itself.